### PR TITLE
inspire_cds_package: update journal mappings

### PR DESCRIPTION
**Pull request by michamos**
* The journal mappings have been cleaned up by @agentilb, and are now
directly read from a csv file that can more easily be updated by curators.

Signed-off-by: Micha Moskovic <michamos@gmail.com>

### CODE REVIEW

**Comment by michamos** [comment]
@kaplun I cannot assign you but would like a review.

File: harvestingkit/inspire_cds_package/journals_mappings.csv [hunk 1]
```diff
@@ -0,0 +1 @@
...
Trans.Amer.Nucl.Soc.,Trans. Am. Nucl. Soc.
Trans.Cambridge Phil.Soc.,Trans. Camb. Philos. Soc.
Transp.Theory Statist.Phys.,Transp. Theory Stat. Phys.
Turk.J.Math,Turk. J. Math.
Twistor Newslett.,Twistor Newsl.
Ukr.Fiz.Zh.(Russ.Ed.),Ukr. Fiz. Zh.
Usp.Fiz.Nauk,Uspekhi Fiz. Nauk
Vak.Tech.,Vak.-Tech.
Verh.Phys.Gesell.,Verh. Deutsch. Phys. Ges
X Ray Spectrom.,X-ray Spectrom.
```
Comment (kaplun) [inline harvestingkit/inspire_cds_package/journals_mappings.csv]: Uh. This is not much checkable. Can you format-it with proper new lines?

File: harvestingkit/inspire_cds_package/from_cds.py [hunk 2]
```diff
@@ -374,8 +374,9 @@ def update_journals(self):
             new_subs = []
             for idx, (key, value) in enumerate(subs):
                 if key == 'p':
-                    new_subs.append((key, self.get_config_item(value,
-                                                               "journals")))
+                    journal_name = self.get_config_item(value, "journals")
+                    journal_name = journal_name.replace('.', '. ').strip()
```
Comment (kaplun) [inline harvestingkit/inspire_cds_package/from_cds.py]: Can you explain this?

**Comment by kaplun** [comment]
BTW you are now admin of this repo too :dancer:

**Comment by michamos** [comment]
I repushed with readable CSV, `pkg_resources` use and your patch.

**Comment by kaplun** [comment]
cc: @switowski

**Comment by kaplun** [comment]
LGTM, beside the test currently failing...

**Comment by michamos** [comment]
@kaplun the test is failing because your patch makes the conversion correct for journals (no substring search), but breaks it for experiments (where substring search is needed because the experiment names are not spelled out fully in the mappings).

**Comment by kaplun** [comment]
OK. I will send a patch to my patch to add a flag to turn the feature on and off :dancer:

**Comment by michamos** [comment]
@kaplun did you ever merge your PR fixing the tests?

**Comment by kaplun** [comment]
Ahem...

**Comment by kaplun** [comment]
Looks like I did it in: f60664e643c0ade243f0141c22aeae4e84642fa2

**Comment by kaplun** [comment]
Closed in #166

**Pull request closed without merging**